app/src/main/java/com/ahargunyllib/growth/presentation/view/authenticated/HistoryDepositScreen.kt

```
@@ -196,7 +288,15 @@ fun FilterDropdown() {
 }
 
 @Composable
-fun HistoryItemCard(item: DepositHistory) {
+fun HistoryItemCard(collection: Collection) {
+    val dateFormat = SimpleDateFormat("dd MMM yyyy", Locale("id", "ID"))
+    val formattedDate = try {
+        val timestamp = collection.createdAt.toLongOrNull() ?: 0L
+        dateFormat.format(Date(timestamp))
+    } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)